### PR TITLE
improve sync doc

### PR DESCRIPTION
**Pull request by zhijian-pro**

### CODE REVIEW

**Comment by zhijian-pro** [comment]
#1542

File: docs/zh_cn/reference/command_reference.md [hunk 1]
```diff
@@ -549,7 +553,7 @@ juicefs sync [command options] SRC DST
 跳过包含 PATTERN (POSIX正则表达式) 的对象名
 
 `--include PATTERN`<br />
-仅同步包含 PATTERN (POSIX正则表达式) 的对象名
+需要配合`--exclude PATTERN`使用，仅同步包含 `include PATTERN` (POSIX正则表达式) 的对象集合中排除包含`exclude PATTERN` (POSIX正则表达式)的对象后的剩余对象集合
```
Comment (davies) [inline docs/zh_cn/reference/command_reference.md:556]: 需要配合`--exclude PATTERN`使用，使得匹配的内容不会被排除而仍然会被拷贝。

File: docs/en/reference/command_reference.md [hunk 2]
```diff
@@ -551,7 +555,7 @@ delete extraneous objects from destination (default: false)
 exclude keys containing PATTERN (POSIX regular expressions)
 
 `--include PATTERN`<br />
-only include keys containing PATTERN (POSIX regular expressions)
+Need to be used with `--exclude PATTERN`. Synchronize only the set of objects remaining after excluding objects containing `exclude PATTERN` (POSIX regular expressions) from the set of objects containing `include PATTERN` (POSIX regular expressions).
```
Comment (davies) [inline docs/en/reference/command_reference.md:558]: Don't exclude the files match with PATTERN

**Comment by zhijian-pro** [comment]
Done in another [PR](https://github.com/juicedata/juicefs/pull/1554).

**Pull request closed without merging**